File: Desafio 1-Classificador de Nível de Herói/desafio1.js
const herois = [
    {nome:"Red", xp:26000},
    {nome:"Blue", xp:10000},
    {nome:"Pokemon", xp:5680},
    {nome:"Serena", xp:3767},
    {nome:"Calem", xp:4821},
    {nome:"Brendan", xp:200},
    {nome:"Lucas", xp:9873}

]

for (let i = 0; i < herois.length; i++) {

let nivel

if (herois[i].xp < 1000) {
    nivel = "ferro"
} else if (herois[i].xp > 1000 && herois[i].xp <= 2000) {
    nivel = "Bronze"
} else if (herois[i].xp > 2000 && herois[i].xp <= 5000) {
    nivel = "Prata"
} else if (herois[i].xp > 5000 && herois[i].xp <= 7000) {
    nivel = "Ouro"
} else if (herois[i].xp > 7000 && herois[i].xp <= 8000) {
    nivel = "Platina"
} else if (herois[i].xp > 8000 && herois[i].xp <= 9000) {
    nivel = "Ascendente"
} else if (herois[i].xp > 9000 && herois[i].xp <= 10000) {
    nivel = "Imortal"
} else if (herois[i].xp >= 10001) {
    nivel = "Radiante"
}

console.log ("O Herói de nome " + herois[i].nome + " está no nível " + nivel)
}

File: Desafio 1-Classificador de Nível de Herói/desafio1.py
def classifies_hero_level(xp_hero):
    if xp_hero <= 1000:
        return "ferro"
    elif xp_hero <= 2000:
        return "bronze"
    elif xp_hero <= 5000:
        return "prata"
    elif xp_hero <= 7000:
        return "ouro"
    elif xp_hero <= 8000:
        return "platina"
    elif xp_hero <= 9000:           
        return "ascendente"
    elif xp_hero <= 10000:
        return "imortal"
    else:
        return "radiante"

def main():
    name_hero = input("Digite o nome do herói: ")
    xp_hero = int(input("Digite a quantidade de XP do herói: "))

    level = classifies_hero_level(xp_hero)
    print(f"O Herói de nome {name_hero} está no nível de {level}") 
    
